### PR TITLE
Tab bar removal from subpages

### PR DESCRIPTION
**Pull request by mateosalta**
Remove bottom new tab bar from pages not related to tab usage, "done" button duplicates current back button function

## History page:
- remove bottom tab bar
- add 'edit' button for better discovery of edit options, and match downloads page
- on domain selection make 'less' button into a close 'x'
- domain selection header height match

screenshot20200720_191623654

screenshot20200720_191626015
screenshot20200720_191458765
screenshot20200720_191511202

## Bookmarks
- remove bottom tab bar
screenshot20200720_191522752
screenshot20200720_191636596

### CODE REVIEW

**Comment by cibersheep** [comment]
Coooool.Coolcoolcool
@mateosalta maybe this as well?: https://github.com/ubports/morph-browser/blob/xenial/src/app/webbrowser/BookmarksViewWide.qml#L56

**Comment by mateosalta** [comment]
oh! I missed that one :)

**Comment by mateosalta** [comment]
And done :)

**Comment by mateosalta** [comment]
Hi @UniversalSuperBox , can this one be considered for ota-15. It goes together well with @cibersheep's color changes, as he didn't change elements to be removed by this request.

the new tab bar was removed from history and bookmarks, as currently they are pages just like the settings and downloads pages, using the new tab button from these pages is confusing ux. the new tab bar will only be displayed on the tab list. 

This also enables us to consider moving these pages to actual tabs, as the new tab page is. if we do that in the future, it will still make sense to not have the tab bar,

File: src/app/webbrowser/BookmarksView.qml [hunk 1]
```diff
@@ -53,45 +53,4 @@ Common.BrowserPage {
             BookmarksModel.remove(url)
         }
     }
-
-    Local.Toolbar {
-        id: toolbar
```
Comment (UniversalSuperBox) [inline src/app/webbrowser/BookmarksView.qml:58]: Removing the toolbar broke anchors to the toolbar in this file.

File: src/app/webbrowser/BookmarksViewWide.qml [hunk 2]
```diff
@@ -52,45 +52,4 @@ Common.BrowserPage {
             BookmarksModel.remove(url)
         }
     }
-
-    Local.Toolbar {
-        id: toolbar
```
Comment (UniversalSuperBox) [inline src/app/webbrowser/BookmarksViewWide.qml:57]: Removing the toolbar broke anchors to the toolbar in this file.

File: src/app/webbrowser/HistoryView.qml [hunk 3]
```diff
@@ -100,50 +100,6 @@ Common.BrowserPage {
         Keys.onDeletePressed: currentItem.removed()
     }
 
-    Local.Toolbar {
-        id: toolbar
```
Comment (UniversalSuperBox) [inline src/app/webbrowser/HistoryView.qml:104]: Removing the toolbar broke anchors to the toolbar in this file.

File: src/app/webbrowser/HistoryViewWide.qml [hunk 4]
```diff
@@ -385,47 +385,6 @@ Common.BrowserPage {
         }
     }
 
-    Local.Toolbar {
-        id: bottomToolbar
```
Comment (UniversalSuperBox) [inline src/app/webbrowser/HistoryViewWide.qml:389]: Removing the bottomToolbar broke anchors to the toolbar in this file.

**Review by UniversalSuperBox — CHANGES_REQUESTED** [review]
Please try installing this change and checking for warnings when loading these pages.

**Comment by mateosalta** [comment]
Ah, sorry @UniversalSuperBox I messed it up with the merge conflict

**Review by mateosalta — COMMENTED** [review]
anchors replaced to not link to toolbar

**Review by UniversalSuperBox — APPROVED** [review]
Changes look good, let's do it.

**Comment by mateosalta** [comment]
nice :) getting more consistant. i do need to work more on the sub-domain page.

**Comment by mateosalta** [comment]
ok, further tweaking of the strange sub page "ExpandedHistoryView" https://github.com/ubports/morph-browser/pull/488

**Pull request merged**